docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+    ListenAddress: 127.0.0.1:8443
+
+    # TLS configuration for the operations endpoint
+    TLS:
+        # TLS enabled
+        Enabled: false
+
+        # Certificate is the location of the PEM encoded TLS certificate
+        Certificate:
+
+        # PrivateKey points to the location of the PEM-encoded key
+        PrivateKey:
+
+        # Most operations service endpoints require client authentication when TLS
+        # is enabled. ClientAuthRequired requires client certificate authentication
+        # at the TLS layer to access all resources.
+        ClientAuthRequired: false
+
+        # Paths to PEM encoded ca certificates to trust for client authentication
+        ClientRootCAs: []
+```
+
+* **`ListenAddress`**: (required when using the operations service) Specify the address and port of the operations server.
+* **`Enabled`**: (required when using the operations service) Must be `true` if the operations service is being used.
+* **`Certificate`**: (required when using the operations service) Can be the same file as the `orderer.tls.cert.file`.
+* **`PrivateKey`**: (required when using the operations service) Can be the same file as the `orderer.tls.key.file`.
+* **`ClientAuthRequired`**: (required when using the operations service) Must be set to `true` to enable mutual TLS between the client and the server.
+* **`ClientRootCAs`**: (required when using the operations service) Similar to the client root CA cert file in TLS, it contains a list of client root CA certificates that can be used to verify client certificates. If the client enrolled with the orderer organization CA, then this value is the orderer organization root CA cert.
+
+## Metrics.Provider.*
+
+By default this is disabled, but if you want to monitor the metrics for the orderer, you need to choose either `StatsD` or `Prometheus` as your metric provider. `StatsD` uses a "push" model, pushing metrics from the orering node to a `StatsD` endpoint. Because of this, it does not require configuration of the operations service itself. See the list of [Available metrics for the orderer](../metrics_reference.html#orderer-metrics).
+
+Because Prometheus utilizes a "pull" model there is not any configuration required, beyond making the operations service available. Rather, Prometheus will send requests to the operations URL to poll for available metrics.
+
+```
+    # The metrics provider is one of statsd, prometheus, or disabled
+    Provider: disabled
+
+    # The statsd configuration
+    Statsd:
+      # network type: tcp or udp
+      Network: udp
+
+      # the statsd server address
+      Address: 127.0.0.1:8125
+
+      # The interval at which locally cached counters and gauges are pushed
+      # to statsd; timings are pushed immediately
+      WriteInterval: 30s
+
+      # The prefix is prepended to all emitted statsd metrics
+      Prefix:
+```
+
+* **`Statsd.Provider`**: Must be `enabled` to use `StatsD` or `Prometheus` metrics for the ordering node.
```

Comment:
I know that lowercase values work. Not sure about the mixed case values unless somebody has mentioned this.

Reply:
I used "StatsD" to refer to StatsD generally (as this is the proper capitalization from what I could find online, but used the capitalization from the YAML when explicitly referring to the YAML.

Reply:
It doesn't matter what case is used online or in the YAML keys. The valid values for Metrics.Provider are:
"disabled"
"statsd"
"prometheus"